### PR TITLE
Fix 118 - Update Android SDK and ADB installation in documentation

### PR DESCRIPTION
**Pull request by EstherPhebe**
closes odk-x/tool-suite-X#118

#### What is included in this PR?
1. Update documentation with the Android SDK and [ADB](https://developer.android.com/studio/command-line/adb.html) installation process with `command-line tools`.

### CODE REVIEW

File: odkx-src/app-designer-prereqs.rst [hunk 1]
```diff
@@ -143,32 +143,41 @@ To install the Android SDK:
 
   .. note::
 
-    You can alternatively install the full Android Studio if you so wish, in which case you should follow Google's instructions and then skip to step 6 of this guide.
+    You can alternatively install the full Android Studio if you so wish, in which case you should follow Google's instructions and .
```
Comment (elmps2018) [inline odkx-src/app-designer-prereqs.rst:146]: there seems to be an "and ." that is confusing.

Comment (EstherPhebe) [reply odkx-src/app-designer-prereqs.rst:146]: I will fix that, didn't catch it before. Thank you.

Comment (elmps2018) [reply odkx-src/app-designer-prereqs.rst]: This reads more clearly now, just going to request that someone more familiar with the development side review!

**Review by wbrunette — CHANGES_REQUESTED** [review]
@EstherPhebe good start. One thing with windows is you have your choice of using "CMD" or "PowerShell". It would be good to update the instructions to refer to both to minimize confusion. PowerShell is newer with more complex features. Either will work just fine in this case so it would be good to recognize this to be more consistent with other ODK-X documentation. Again I suggest changing it so that you say CMD or PowerShell.

**Review by elmps2018 — COMMENTED** [review]
This revision looks good to me, @wbrunette can you check that this addressed your comment?

**Review by wbrunette — APPROVED** [review]

**Pull request merged**